docs/section19/19-19.md

```diff
@@ -0,0 +1,96 @@
+            - Get the total relief cfm, need to get the four element in the list of fan CFMs: `total_relief_cfm = fan_sys_CFM[3]`  
+
+            - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                    - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                    - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`                    
+            
+            Below determines the expected fan power:
+            - Check if there are any energy recovery objects associated with the fan system object: `if fan_sys_b.air_energy_recovery != Null:` 
+                - Get the enthalpy recovery ratio: `enthalpy_rec_ratio = fan_sys_b.air_energy_recovery.enthalpy_recovery_ratio`  
+                - Assume that the outdoor air flow through the ERV is equal to the minimum outdoor airflow: `ERV_OA_air_flow = fan_sys_b.minimum_outdoor_airflow`  
+                - Assume that the exhaust air flow through the ERV is equal to the maximum of either the total exhaust cfm or total relief cfm associated with the fan system: `ERV_Ex_air_flow = max(total_exhaust_cfm,total_relief_cfm)`  
+                - Calculate A per Section 6.5.3.1.1: `A = (((2.2*enthalpy_rec_ratio)-0.5) *(ERV_Ex_air_flow + ERV_OA_air_flow))/4131`  
+            
+            - Check if the air_filter_merv_rating was populated: `if fan_sys_b.air_filter_merv_rating != Null:`  
+                - Get the MERV filter rating: `MERV_rating = fan_sys_b.air_filter_merv_rating`  
+            - Else: `Else: MERV_rating = 0`  
+            - Determine the pressure drop adjustment in in. of water if MERV is 9 through 12: `if MERV_rating >= 9 and MERV_rating <=12: MERV_adj = 0.5`  
+            - Else if, MERV is 13 and above (assume 0.9 for all MERVs greater than 12 since we do not know the pressure drop for MERV 16 and greater): `elif MERV_rating >12: MERV_adj = 0.9`  
+            - Else: `Else: MERV_rating = 0`  
+            - Calculate A per Section 6.5.3.1.1 for MERV assuming cfmd equals the supply cfm: `A = A + (MERV_adj * supply_cfm_b/4131)`  
+            
+            - Create a list of the CV system types: `CV_system_types = [HVAC_SYS.SYS_3, HVAC_SYS.SYS_4,HVAC_SYS.SYS_12, HVAC_SYS.SYS_13]`  
+            - Check if system type is constant volume:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in CV_system_types):`     
+                - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.00094 * supply_cfm_b) + A`  
```

Comment:
Recommend defining minimum_BHP and minimum_fan_power for each system type which would be based on A=0. Then we can do FAIL outcome if the modeled baseline fan power is lower than this minimum.

Reply:
Sounds good. Need to implement.

Reply:
Implemented.